Zend/zend.c

```diff
@@ -1619,14 +1622,35 @@ ZEND_API ZEND_COLD void zend_error_zstr(int type, zend_string *message) {
 	zend_error_zstr_at(type, filename, lineno, message);
 }
 
-ZEND_API void zend_begin_record_errors(void)
+static inline void zend_begin_record_errors_ex(bool no_emmitting)
```

Comment:
would `zend_always_inline` be inappropriate ?

Reply:
I guess it's better for consistency as in this case it won't change much perf wise... Anyway changed.